data/human/hails.txt

```diff
@@ -1982,24 +1974,158 @@ phrase "friendly civilian"
 		"Don't get on the bad side of the authorities. "
 		"I always try to be an honest captain. "
 		"Crime doesn't pay in the end. "
-		""
+		"You've got to be careful if you associate yourself with the wrong sorts. "
+		"" 4
 	word
 		"It's a fact"
 		"I've heard"
-		"I heard a rumor"
 		"You should know"
 		"They say"
 		"It's said"
+		"I know from experience"
+		"Once you've been on the starlanes long enough you know"
 	word
 		" that "
+		", "
```

Comment:
This comma feels kind of weird and unnatural. An example sentence: "Once you've been on the starlanes long enough you know, smuggling is a sure way to find yourself behind bars."

Reply:
I'll need to look at this hail again.

Reply:
Okay, using `replace` I should have fixed it.